### PR TITLE
Seperate CUI url paths and tests

### PR DESCRIPTION
**Pull request by matmair**
This PR creates a clearer separation between CUI and PUI URLs, enabling users to fully disable CUI content serving.  It also tags CUI test to make it possible to disable running them temporarily.

### CODE REVIEW

File: InvenTree/order/test_views.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 """Unit tests for Order views (see views.py)."""
 
+from django.conf.locale import ta
```
Comment (SchrodingersGat) [inline InvenTree/order/test_views.py:3]: Is this line an error?

File: InvenTree/stock/models.py [hunk 2]
```diff
@@ -2304,7 +2309,9 @@ def get_api_url():
 
     def get_absolute_url(self):
         """Return url for instance."""
-        return f'/stock/track/{self.id}'
+        if settings.ENABLE_CLASSIC_FRONTEND:
+            return f'/stock/track/{self.id}'
+        return InvenTree.helpers.pui_url(f'/stock/id/{self.item.id}')
```
Comment (SchrodingersGat) [inline InvenTree/stock/models.py:2314]: I think this pui_url is incorrect?

**Review by SchrodingersGat — COMMENTED** [review]
Overall pretty reasonable, a bit of code to prune back out later on but should be OK

File: InvenTree/order/test_views.py (hunk 1 above)
Comment (matmair) [reply InvenTree/order/test_views.py:3]: yes, fixed now

File: InvenTree/stock/models.py (hunk 2 above)
Comment (matmair) [reply InvenTree/stock/models.py:2314]: You are right, fixed now

**Comment by matmair** [comment]
@SchrodingersGat I hope to just remove most code that is tagged at that point ;-)

**Comment by SchrodingersGat** [comment]
Sounds good. Happy to merge this when it passes?

**Comment by matmair** [comment]
Yes please @SchrodingersGat

**Pull request merged**